src/server/utils/onModuleLoad.js

```diff
@@ -108,6 +110,9 @@ export default function onModuleLoad({
     if (provideStateConfig) {
       setStateConfig(provideStateConfig);
     }
+    if (pwa) {
+      configurePWA(pwa);
+    }
```

Comment:
else condition? (ex: going from a version that configures a Service Worker to a version that doesn't, should the service worker be kept from the previous version or removed?)

Reply:
Solid point, I've made it a catchall so it always runs and handles that case.